Don't limit faction cubes to hero in end of combat (#1064)
Heegu-sama/Homm3BG: sections/combat.tex

\end{multicols*}

\pagetarget{Endcombat}{\subheader{End of Combat}}

> \begin{multicols}{2}

Reviewer: You sure about this? This gap at the bottom is all to noticeable.

A \textbf{Winning Hero} gets \pagelink{Combatexperience}{Experience}, if it was the Main Hero. If it was the attacker, they must visit the Field where the Combat took place.\par
A \textbf{Retreating Hero} from a Combat with Neutral Units has to move back to the last visited Field on the map (without visiting that Field).\par

> A \textbf{defeated Main Hero} has to move to a friendly Town or Settlement. If they were defeated by another player's army (with or without hero), they gain 1~\svg{morale_negative} and have to pay 5 \svg{gold} to the winner. Furthermore, they give a \pagelink{End}{Faction Cube} to the winning player. Defeating a Main Hero may cause \pagelink{End}{Player Elimination}.\par

Reviewer: ```suggestion
A \textbf{defeated Main Hero} has to move to a friendly Town or Settlement. If they were defeated by another player's army (with or without a Hero), they gain 1~\svg{morale_negative} and have to pay 5 \svg{gold} to the winner. Furthermore, they give a \pagelink{End}{Faction Cube} to the winning player. Defeating a Main Hero may cause \pagelink{End}{Player Elimination}.\par
```

A \textbf{Winning Hero} gets \pagelink{Combatexperience}{Experience}, if it was the Main Hero. If it was the attacker, they must visit the Field where the Combat took place.\par
A \textbf{Retreating Hero} from a Combat with Neutral Units has to move back to the last visited Field on the map (without visiting that Field).\par

> A \textbf{defeated Main Hero} has to move to a friendly Town or Settlement. If they were defeated by another player's Hero, Town or Settlement, they gain 1~\svg{morale_negative} and have to pay 5 \svg{gold} to the winner. Furthermore, they give a \pagelink{End}{Faction Cube} to the winning player. Defeating a Main Hero may cause \pagelink{End}{Player Elimination}.\par

Reviewer 2: What about this? 
```suggestion
A \textbf{defeated Main Hero} has to move to a friendly Town or Settlement. If they were defeated by another player's army (with or without hero), they gain 1~\svg{morale_negative} and have to pay 5 \svg{gold} to the winner. Furthermore, they give a \pagelink{End}{Faction Cube} to the winning player. Defeating a Main Hero may cause \pagelink{End}{Player Elimination}.\par
```

OR even shorter:
```suggestion
A \textbf{defeated Main Hero} has to move to a friendly Town or Settlement. If they were defeated by another player, they gain 1~\svg{morale_negative} and have to pay 5 \svg{gold} to the winner. Furthermore, they give a \pagelink{End}{Faction Cube} to the winning player. Defeating a Main Hero may cause \pagelink{End}{Player Elimination}.\par
```

Author: I tried to make it clear that fight against neutrals which are controlled by a PLAYER don't trigger this. But "player's army" probably conveys that too, yeah

> A \textbf{defeated Secondary Hero} (even instantly defeated without combat) is removed from the map. If defeated by another player's Hero/Town/Settlement they also get a Faction Cube.\par

Reviewer: Town as in units without a Hero? How can a settlement defeat you? Maybe it should be explicit about besieging.

Author: if your hero attacked an enemy settlement, and the enemy player paid the gold to transport army there, and that army defeated your hero

Reviewer: ```suggestion
A \textbf{defeated Secondary Hero} (even instantly defeated without combat) is removed from the map. If defeated by another player's Hero, or upon losing a siege against their Town or Settlement, that player also gets a Faction Cube.\par
```
The way it's written now it sounds like the Town or Settlement is attacking, while it's the other way around. I mean something like this?

> A \textbf{defeated Secondary Hero} (even instantly defeated without combat) is removed from the map. If defeated by another player's Hero/Town/Settlement they also get a Faction Cube.\par

Reviewer 2: Here too, maybe shorter:
```suggestion
A \textbf{defeated Secondary Hero} (even instantly defeated without combat) is removed from the map. If defeated by another player's army they also get a Faction Cube.\par
```

A \textbf{Surrendering Hero} doesn't count as defeated. Move a surrendering Main Hero to a town or settlement, or remove a Secondary Hero from the map as if they had been defeated. The other player neither gets Experience nor Faction Cubes.\par

> If your \textbf{Faction Town} is defeated, give the winner a Faction Cube (in addition to the winner flagging this Town).\par

Reviewer: Can this be combined with the siege loss? Single-sentence paragraphs don't look good, especially more than one in a row.

Author: Combined how?

Reviewer: Nothing, I've reconsidered.

> If your \textbf{Faction Town} is defeated, give the winner a Faction Cube (in addition to the winner flagging this Town).\par

Reviewer 2: I would delete the "in addition", cause by the rules it is irrelevant to collect more than one faction cube per player. The victory condition is to get one faction cube from every other player. doenst make sense to collect two. 
```suggestion
If your \textbf{Faction Town} is defeated, give the winner a Faction Cube.\par
```
